pages/hoax-for-you.js

```diff
@@ -0,0 +1,44 @@
+import ArticlePageLayout from 'components/ArticlePageLayout';
+import { useRouter } from 'next/router';
+import Head from 'next/head';
+import getConfig from 'next/config';
+import { t } from 'ttag';
+import querystring from 'querystring';
+import AppLayout from 'components/AppLayout';
+
+const {
+  publicRuntimeConfig: { PUBLIC_URL },
+} = getConfig();
+
+function HoaxForYouPage() {
+  const { query } = useRouter();
+  const queryString = querystring.stringify(query);
+
+  return (
+    <AppLayout>
+      <Head>
+        <title>{t`Hoax for you`}</title>
+        <link
+          rel="alternate"
+          type="application/rss+xml"
+          href={`${PUBLIC_URL}/api/articles/rss2?${queryString}`}
```

Comment:
I think we should remember to revisit RSS feed URLs later (in future PRs), because currently these `queryString` does not reflect the actual query.